pyfr/polys.py

```diff
@@ -32,7 +31,7 @@ def jacobi(n, a, b, z):
 
 
 def jacobi_diff(n, a, b, z):
```

Comment:
Again, I think you might need some checks on the validity of a and b, and some defaults would be good.

Reply:
Defaulting would be difficult as it would force a and b to be at the end which would be somewhat non-standard.

Reply:
Ok, I still think you should add tests for the validity of a and b though

Reply:
I think the below is valid;
`def jacobi_diff(n, a=0, b=0, *, z):`
but this forces 'z' to be a keyword only argument which might be inconvenient.